package.json

```diff
@@ -1,50 +0,0 @@
-{
-  "name": "fathom-product",
-  "description": "Product extraction using Fathom.",
-  "version": "0.1.0",
-  "author": "Swathi Iyer <siyer@mozilla.com>",
-  "bugs": {
-    "url": "https://github.com/swathiiyer2/fathom-products/issues"
-  },
-  "dependencies": {
-    "fathom-web": "^2.1.0",
-    "object-hash": "^1.1.8",
```

Comment:
I don't see anywhere you're actually using wu or object-hash. Did I miss it? (You don't need to include "deep" dependencies here; it will automatically pull in "wu" because it's a fathom-web dep.)